src/hivpy/hiv_status.py

```diff
@@ -173,6 +226,462 @@ def update_partner_risk_vectors(self, population: Population):
+                                                              COND(col.AGE, op.lt, 65),
+                                                              COND(col.ON_ART, op.eq, True),
+                                                              COND(col.VIRAL_SUPPRESSION, op.eq, True))))
+        num_on_art = len(population.get_sub_pop(AND(COND(col.AGE, op.ge, 15),
+                                                    COND(col.AGE, op.lt, 65),
+                                                    COND(col.ON_ART, op.eq, True))))
+        self.proportion_viral_suppressed = safe_ratio(num_viral_suppressed, num_on_art)
+
+        num_ltp_viral_suppressed = len(population.get_sub_pop(AND(COND(col.AGE, op.ge, 15),
+                                                                  COND(col.AGE, op.lt, 65),
+                                                                  COND(col.LTP_ART, op.eq, True),
+                                                                  COND(col.LTP_VIRAL_SUPPRESSED, op.eq, True))))
+        num_ltp_on_art = len(population.get_sub_pop(AND(COND(col.AGE, op.ge, 15),
+                                                        COND(col.AGE, op.lt, 65),
+                                                        COND(col.LTP_ART, op.eq, True))))
+        self.proportion_ltp_viral_suppressed = safe_ratio(num_ltp_viral_suppressed, num_ltp_on_art)
+        self.diff_proportion_viral_suppressed = self.proportion_viral_suppressed - self.proportion_ltp_viral_suppressed
+
+    # TODO: Probably move to ART module when it is ready
+    def update_art_stats(self, population):
+        num_diagnosed_on_art = len(population.get_sub_pop(AND(COND(col.HIV_DIAGNOSED, op.eq, True),
+                                                              COND(col.ON_ART, op.eq, True),
+                                                              COND(col.AGE, op.ge, 15),
+                                                              COND(col.AGE, op.lt, 65))))
+        num_diagnosed = len(population.get_sub_pop(AND(COND(col.HIV_DIAGNOSED, op.eq, True),
+                                                       COND(col.AGE, op.ge, 15),
+                                                       COND(col.AGE, op.lt, 15))))
+        self.proportion_diagnosed_on_art = safe_ratio(num_diagnosed_on_art, num_diagnosed)
+
+        num_ltp_diagnosed_on_art = len(population.get_sub_pop(AND(COND(col.LTP_DIAGNOSED, op.eq, True),
+                                                                  COND(col.LTP_ART, op.eq, True),
+                                                                  COND(col.AGE, op.ge, 15),
+                                                                  COND(col.AGE, op.lt, 65))))
+        num_ltp_diagnosed = len(population.get_sub_pop(AND(COND(col.LTP_DIAGNOSED, op.eq, True),
+                                                           COND(col.AGE, op.ge, 15),
+                                                           COND(col.AGE, op.lt, 15))))
```

Comment:
```suggestion
        num_diagnosed = len(population.get_sub_pop(AND(COND(col.HIV_DIAGNOSED, op.eq, True),
                                                       COND(col.AGE, op.ge, 15),
                                                       COND(col.AGE, op.lt, 65))))
        self.proportion_diagnosed_on_art = safe_ratio(num_diagnosed_on_art, num_diagnosed)

        num_ltp_diagnosed_on_art = len(population.get_sub_pop(AND(COND(col.LTP_DIAGNOSED, op.eq, True),
                                                                  COND(col.LTP_ART, op.eq, True),
                                                                  COND(col.AGE, op.ge, 15),
                                                                  COND(col.AGE, op.lt, 65))))
        num_ltp_diagnosed = len(population.get_sub_pop(AND(COND(col.LTP_DIAGNOSED, op.eq, True),
                                                           COND(col.AGE, op.ge, 15),
                                                           COND(col.AGE, op.lt, 65))))
```
All these age ranges should be <65.